Vagrantfile

```diff
@@ -12,6 +12,7 @@ $script = <<SCRIPT
 cd /home/vagrant/oppia
 bash ./scripts/install_prerequisites.sh
 bash ./scripts/start.sh
+touch /etc/is_vagrant_vm
```

Comment:
Can you add a comment here referring to scripts/setup.sh, so that other developers know what this is for?